e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -1,21 +1,25 @@
 package uk.nhs.adaptors.gp2gp.e2e;
 
 import static org.assertj.core.api.Assertions.assertThat;
-
 import static uk.nhs.adaptors.gp2gp.e2e.AwaitHelper.waitFor;
 
 import java.nio.charset.Charset;
 import java.util.Collections;
 import java.util.UUID;
 
 import org.apache.commons.io.IOUtils;
+import org.assertj.core.api.SoftAssertions;
+import org.assertj.core.api.junit.jupiter.InjectSoftAssertions;
 import org.bson.Document;
 import org.junit.jupiter.api.Test;
 
 import uk.nhs.adaptors.gp2gp.MessageQueue;
 import uk.nhs.adaptors.gp2gp.Mongo;
 
 public class EhrExtractTest {
+    @InjectSoftAssertions
+    private final SoftAssertions softly = new SoftAssertions();
```

Comment:
I don't think this is correct usage of the InjectSoftAssertions annotation

https://assertj.github.io/doc/